src/components/logs/Line.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, ListItem, Stack } from '@mui/material';
+import { parse } from 'ansicolor';
+import { ViewLogs_Line } from 'types';
+import LinePart from './LinePart';
+
+interface Props {
+    line: ViewLogs_Line | string;
+    lineNumber: number | string | any;
+    disableSelect?: boolean;
+}
+
+export const lineNumberColor = '#666';
+
+// const parseStream = (line: ViewLogs_Line): ParsedStream => {
+//     return line.stream.slice(0, line.stream.lastIndexOf(':')) as ParsedStream;
+// };
+
+function LogLine({ line, lineNumber, disableSelect }: Props) {
+    let parsedLine;
+
+    if (line instanceof Object) {
+        parsedLine = parse(line.log_line);
+    } else {
+        parsedLine = parse(line);
+    }
+
+    return (
+        <ListItem
+            sx={{
+                'userSelect': disableSelect ? 'none' : undefined,
+                'py': 0,
+                '&:hover': {
+                    background: (theme) =>
+                        theme.palette.mode === 'dark' ? '#222' : '#eee',
+                },
```

Comment:
Rando colors that looked good enough for now. Not making these official yet as we need to basically now design all the "logger" stuff since monaco won't do it for free anymore.